doc/api/esm.md

```diff
@@ -1166,8 +1165,7 @@ console.log(five); // Logs `5`
 ```
 
 ```bash
-node b.mjs # fails
-node --experimental-top-level-await b.mjs # works
+node b.mjs # works
```

Comment:
Example here shows how to use the flag and that the flag is important for this feature to work. But since it's now expected by default, maybe this example is redundant?

Reply:
I think that this documentation is still useful to explain that we support TLA and to clearly mark it as experimental